docs/configuration.asciidoc

```diff
@@ -290,19 +290,58 @@ storage use in your Elasticsearch cluster.
 
 |============
 | Environment                            | `Config` key               | Default
-| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `5`
+| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `"5ms"`
 |============
 
-Use this to disable stacktrace collection for spans with a duration shorter than or equal to the given amount of milleseconds.
+Use this to disable stacktrace collection for spans with a duration shorter
+than or equal to the given amount of milleseconds.
```

Comment:
I suggest to reuse the term _frames_ from the actual config option in the description to be more consistent. (Instead of using _collection_).